rag_eval/evals.py

```
@@ -0,0 +1,914 @@
+
+    def create_evaluators(self):
+        """
+        Create evaluator functions for run_experiment.
+
+        Returns:
+            List of evaluator functions
+        """
+        evaluators = []
+
+        # Evaluator for average dataset relevance
+        def avg_dataset_relevance_evaluator(*, output, **kwargs):
```

Ruff (0.15.1)

[warning] Unused function argument: `kwargs`

(ARG001)